source/parser.hera

```diff
@@ -608,6 +605,20 @@ CallExpression
 
     return member
 
+# NOTE: Special `return.value` (and `return =` shorthand)
+# for changing the automatic return value of function
+ReturnValue
+  ( "return.value" NonIdContinue ) / ( Return &AfterReturnShorthand ) ->
+    return { type: "ReturnValue", children: [$1[0]] }
+
+AfterReturnShorthand
+  WAssignmentOp
+  UpdateExpressionSymbol
+  # NOTE: Colon indicates potential type annotation, e.g. `let return: T`
+  _? Colon
```

Comment:
This could be `ReturnTypeSuffix`. There isn't really any downside to being more explicit.

Reply:
Given that the context here is a `LexicalDeclaration`, I think `TypeSuffix` is appropriate. Agreed?

Reply:
👍